src/create_py_project/scaffold.py

```
@@ -0,0 +1,96 @@
+    _run(
+        ["git", "commit", "-m", "chore: initial scaffold from create-py-project"],
+        cwd=project_dir,
+    )
+
+    # GitHub
+    push_choice = questionary.select(
+        "Push to GitHub?", choices=["public", "private", "skip"], default="skip"
+    ).ask()
+    if push_choice in ("public", "private"):
+        console.print("[dim]Creating GitHub repo...[/dim]")
+        subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)